src/SmartComponents/Advisor/Advisor.js

```diff
@@ -60,12 +56,17 @@ const Advisor = ({ recStats, recStatsStatus, advisorFetchStatsRecs, advisorFetch
     const pieLegendData = categoryData.map(item => ({ name: `${item.y} ${item.x} `, symbol: { fill: `${item.fill} `, type: 'square' } }));
 
     useEffect(() => {
-        const options = { ...selectedTags.length && ({ tags: selectedTags.join() }), ...(workloads && workloadQueryBuilder(workloads)) };
+        const sapFilter = generateFilter({
```

Comment:
Would it be worthwhile to move `sapFilter` into the common utils file?

Reply:
hmmmm well every card that supports, will be using, and yah see we do the same thing in both patch and inv card... lemme give it a try n <3